templates/sw.tpl.js

```diff
@@ -1,18 +1,4 @@
-
+self.importScripts("lib/sw-toolbox/sw-toolbox.js");
```

Comment:
`importScripts` may delay entire lifecycle, and it blocks the execution of subsequent code.

Reply:
Another network request also may delay entire lifecycle of service worker.